tests/test_file_upload.py

```
@@ -0,0 +1,130 @@
+    profile = AttackProfile(
+        name="File Upload Bypass",
+        category="file_upload",
+        severity="high",
+        description="test",
+        payloads=["test.php"],
+        target_fields=["file", "upload", "avatar"],
+        success_indicators={}
+    )
+    planner.attack_profiles = [profile]
+    
+    attacks = planner.plan_attacks(endpoint)
```

Ruff (0.15.0)

[error] Local variable `attacks` is assigned to but never used

Remove assignment to unused variable `attacks`

(F841)